docs/agile-development/delivery-plan/README.md

```diff
@@ -0,0 +1,50 @@
+# Delivery Plan
+
+## Goals
+
+While Scrum does not require, and in fact discourages planning more than one sprint at a time, most of us work in enterprises where we are dependent on each other. For example: marketing, sales, and support needs to have know what is coming up so they can make plans for budget, people and resources. A delivery plan helps to ensure your teams are aligning with your organizational goals.
```

Comment:
Nit: would suggest using "Agile" vs "Scrum" as different teams tend to use different versions of Agile, not just Scrum
```suggestion
While Agile does not require, and in fact discourages planning more than one sprint at a time, most of us work in enterprises where we are dependent on each other. For example: marketing, sales, and support needs to have know what is coming up so they can make plans for budget, people and resources. A delivery plan helps to ensure your teams are aligning with your organizational goals.
```